application/values-test.yaml

```diff
@@ -230,6 +231,8 @@ persistence:
   annotations:
   #  "helm.sh/resource-policy": keep
   storageSize: 8Gi
+  volumeMode: ""
+  volumeName: ""
```

Comment:
better to give some name and mode in values-test to validate these fields in pipeline